src/antares/tools/ini_tool.py

```diff
@@ -132,6 +145,9 @@ def update_from_ini_file(self) -> None:
 
         self._ini_contents = parsed_ini
 
+    def update_binding_constraints_from_ini_file(self) -> None:
```

Comment:
You don't use this method, you can remove it

Reply:
you are right!